### PR TITLE
Basic CSS practice

### PR DESCRIPTION
**Pull request by Jalaj2705**
Classes and IDs

### CODE REVIEW

File: CSS/Cascaded style sheet.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html>
+
```
Comment (Kunal0215) [inline CSS/Cascaded style sheet.html:3]: remove balnk white spaces

File: CSS/Cascaded style sheet.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+</style>
+
+</head>
+<body>
+
+	<p class="red large">The quick brown fox jumped over the lazy dog.</p>
+
```
Comment (Kunal0215) [inline CSS/Cascaded style sheet.html:28]: this also

File: Hello world.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<html>
+<body>Hello World</body>
```
Comment (Kunal0215) [inline Hello world.html:2]: indentation

**Review by Kunal0215 — CHANGES_REQUESTED** [review]
Fix those above mentioned

File: CSS/Cascaded style sheet.html (hunk 1 above)
Comment (Jalaj2705) [reply CSS/Cascaded style sheet.html:3]: this is necessary

Comment (Kunal0215) [reply CSS/Cascaded style sheet.html:3]: ok

**Pull request merged**